k-distribution/tests/regression-new/map-symbolic-tests/remove

```diff
@@ -0,0 +1,19 @@
+//copied from lookup test plan
+
```

Comment:
Get rid of these two lines, I don't think the `gen-tests.sh` script will treat them correctly.

Also, you need to modify `gen-tests.sh` to also call `remove` in addition to `assignment lookup inkeys`.

Reply:
The test files `remove-#-spec.k` should be populated by the `gen-tests.sh` script here.

Reply:
I didn't use the script. What is it supposed to do?

Reply:
Done.